src/04-modality-specific-files/05-task-events.md

```diff
@@ -102,20 +102,27 @@ onset duration  trial_type  identifier  database  response_time
 5.6 0.6 sad AF01ANSA  kdef  1.739
 ```
 
-This file should be accompanied by a data dictionary.  
+This file should be accompanied by a data dictionary.
 
 ```Text
 sub-control01/
     func/
         sub-control01_task-emoface_events.json
 ```
 
-The `trial_type` and `identifier` columns from the `*_events.tsv` files might be described in this
-dictionary as follows. 
-Note that all other columns SHOULD also be described but are omitted for the sake
-of example.
+Specific fields related to stimulus presentation details SHOULD also be present in this data dictionary:
 
-```JSON  
+| Column name                  | Description                                                                                                                                                                                                                      |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StimulusPresentationOS       | RECOMMENDED. Operating system used to run the stimuli presentation software.                                                                                                                                                     |
```

Comment:
it would be great if we could either

1. standardize this field or
1. provide explicit and simple examples on how to report for the major OSses (Linux, Windows, MacOS)

just so that we don't end up with fields like `StimulusPresentationOS: 'linux'`, and instead get something more detailed (like you also provide in your example): `StimulusPresentationOS: 'Linux-Ubuntu 18.04.5 LTS'`

We'd have to agree what level of detail would be good to have, and probably that level of detail is different for each OS, so I am tending towards my suggestion 2., instead of 1.

Reply:
Will try to provide examples but I am getting the feeling that suggestion 2 might turn into a slippery slope towards suggestion 1. :rofl:

Reply:
Tried to stick to somethng "simple" enough but also flexible for describing the OS part. LMKWYT